spec/changelog.yml

```
@@ -2,6 +2,11 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
+- version: 2.6.2
+  changes:
+  - description: Add `ignore_malformed` to data_stream/fields/fields.spec.yml
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/498
 - version: 2.6.1-next
   changes:
   - description: Prepare for next version
```

This is a new feature, please start a new minor and remove the block for 2.6.1.
```suggestion
- version: 2.7.0-next
  changes:
  - description: Add `ignore_malformed` to data_stream/fields/fields.spec.yml
    type: enhancement
    link: https://github.com/elastic/package-spec/pull/498
```